nymea-plugins-modbus.pro

```diff
@@ -2,7 +2,7 @@ TEMPLATE = subdirs
 
 # Note keep it ordered so the lib will be built first
 CONFIG += ordered
```

Comment:
every time you use `CONFIG += ordered` a kitten dies :-P

https://blog.rburchell.com/2013/10/every-time-you-configordered-kitten-dies.html

better using
`libnymea-sunspec.depends = libnymea-modbus` 
instead, that is, if the sunspec depends on the modbus lib.

The reason is that config+=ordered disables parallell building

Reply:
woops, must be an old copy past error...

Reply:
Will be fixed in the #66 for better testing purpose.

Reply:
https://github.com/nymea/nymea-plugins-modbus/pull/66/commits/e4a42817102e761c6d39b52faf761a853fcdd58a